### ISSUE TITLE
Remove `src/components/configurationlib/deprecated.ts`

### ISSUE THREAD

**Issue by tamuratak** [James-Yu/LaTeX-Workshop#3209]
I am not interested in maintaining `src/components/configurationlib/deprecated.ts`. How about removing `src/components/configurationlib/deprecated.ts`? 

It would be enough to notify deprecated configurations in `CHANGELOG.md`

**Comment by jlelong** [comment]
I am fine with notifying deprecated configurations in `CHANGELOG.md`. However, I doubt that many users read the changelog, what about automatically show the content of `CHANGELOG.md` on update?

**Comment by tamuratak** [comment]
I am not interested in making an effort for users, especially on this kind of thing. Users can learn that they had better read `CHANGELOG.md` after updates. It is enough.

**Comment by jlelong** [comment]
I do not think this is fair to blame users for not reading the `CHANGELOG.md`. This is often considered as intended for developers to keep track of the most important changes. Moreover, the extensions are often updated in the background and the users may not even notice an update. But fine, I will try to find some time to look at this.

**Comment by tamuratak** [comment]
We don't have to care about users so much.

**Comment by James-Yu** [comment]
It's the project maintaining philosophy. I myself is with @jlelong and we need to care about user to some extent, for reasonable voices. It is somehow unreasonable to request users to read the technical details of a plugin before using it. At least I do not check carefully changelogs when I encounter bugs in other plugins. Maybe I am also a not-so-good user and I fully accept the fact.

However, I do think we need to re-think how we handle user voices, for @jlelong and @tamuratak (and for the lazy repo owner @James-Yu who did nothing in the past two years). We don't need to pose us at a position that is against the users (voice). They in most cases are ones that are lack of knowledge on the structure of this plugin or even how to properly file an issue. Also, in most cases they are not hostile, unfriendly, or provocative. We also do not need to over-fiercely counteract. It is very true that some user voices are provoking. Then close the issue, lock the user (with caution), and forget about it.

This attitude is somehow good to the community. But I'd humbly emphasize that, we the project maintainer, are those who benefit the most. You successfully avoid taking bad mood from the cyber space into your real life. LW is a coding project, and it is just a coding project. We have enough moral level to make it good. But it is always just a coding project. Don't like to maintain some part of the code, leave it be (and silently hoping others may work on that though not quite possible). Don't like any people in the context, ignore her. This is also the case I observed in huge projects like Linux and VS Code. We can't make everyone happy. But we can ignore things that may make ourselves unhappy.

Hope the not-native-English words may ease any of you.
